### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Adding new nav button named "Trends" for current trends in software field

### DISCUSSION THREAD

**Discussion by mostafasobh** [dailydotdev/daily#1275] [Feature Requests]
### Description

I suggest to have that nav button which will make the extension super attractive for people who wants to read about latest trends in the software field.

### Solution Proposed

i don't have it yet, but i would be more than happy to contribute the solution, if it meets your approval

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @mostafasobh,

Thanks for the suggestion!
